libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/ActiveGuidanceScreen.kt

```diff
@@ -1,31 +1,28 @@
 package com.mapbox.androidauto.navigation
 
 import androidx.car.app.Screen
-import androidx.car.app.model.Action
-import androidx.car.app.model.ActionStrip
 import androidx.car.app.model.CarColor
 import androidx.car.app.model.Template
 import androidx.car.app.navigation.model.NavigationTemplate
 import androidx.lifecycle.DefaultLifecycleObserver
 import androidx.lifecycle.LifecycleOwner
 import com.mapbox.androidauto.MapboxCarContext
-import com.mapbox.androidauto.R
-import com.mapbox.androidauto.action.MapboxActionProvider
 import com.mapbox.androidauto.action.MapboxMapActionStrip
 import com.mapbox.androidauto.internal.logAndroidAuto
 import com.mapbox.androidauto.location.CarLocationRenderer
 import com.mapbox.androidauto.navigation.roadlabel.CarRoadLabelRenderer
 import com.mapbox.androidauto.navigation.speedlimit.CarSpeedLimitRenderer
 import com.mapbox.androidauto.preview.CarRouteLineRenderer
+import com.mapbox.androidauto.screenmanager.MapboxScreen
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
 import com.mapbox.navigation.core.lifecycle.MapboxNavigationApp
 
 /**
  * After a route has been selected. This view gives turn-by-turn instructions
  * for completing the route.
  */
 internal class ActiveGuidanceScreen constructor(
-    private val mapboxCarContext: MapboxCarContext,
-    private val actionProviders: List<MapboxActionProvider>,
```

Comment:
Is there a reason to have `MapboxActionProvider` interface now?

Reply:
Looking into it there are two of them at this time. `CarAudioGuidanceUi` and `CarFeedbackAction`, they reduce some duplicate code. There could be alternatives but I'm not sure if it's worth changing

Reply:
It looks like we can keep these two classes, but remove the interface, because it is not used anywhere on its own.

Reply:
Trying to think of reasons why we may want to keep it..

Am deleting, we can follow [You aren't gonna need it](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)